razzle.config.js

```diff
@@ -0,0 +1,21 @@
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+module.exports = {
+  modify(config, { target, dev }, webpack) {
+    const appConfig = config; // stay immutable here
+
+    if (target === 'node' && dev) {
+      appConfig.plugins = [
+        new BundleAnalyzerPlugin({
+          analyzerMode: 'static',
+          generateStatsFile: true,
```

Comment:
We may have a use-case for putting a static HTML or JSON file in the future, but I don't think there's value in it right now. You can serve the analyser over a port so maybe we should just do that.

Reply:
I had problems with serving it via a port.  The page would load as expected at http://127.0.0.1:8888, but then as soon as I visited `http://localhost:3000/` for the app, the cli gave me an error `listen EADDRINUSE 127.0.0.1:8888` and the app stopped running. 

I don't believe this analyser was set up for use in this way by a razzle app, which has both client and server webpack configurations.

Reply:
You can change the port by setting `analyzerPort` in the `options` object.